res/skins/LateNight/style_palemoon.qss

```diff
@@ -2498,20 +2504,32 @@ QPushButton#pushButtonRepeatPlaylist:!checked {
 /* AutoDJ button icons */
 
 /* widgets in cue popup menu */
-WCueMenuPopup #CueDeleteButton {
-  qproperty-icon: url(skins:LateNight/palemoon/buttons/btn__delete.svg);
+WCueMenuPopup #CueDeleteButton,
+WCueMenuPopup #CueSavedLoopButton {
   width: 24px;
   height: 42px;
-  /* make the icon slightly larger than default 16px */
-  qproperty-iconSize: 20px;
 }
+WCueMenuPopup #CueDeleteButton {
+  /* Note: we can't use qproperty-icon here because it's evauated only once
+     and therefore won't style the checked state */
+  image: url(skins:LateNight/palemoon/buttons/btn__delete.svg);
+}
+  WCueMenuPopup #CueDeleteButton:pressed {
+    background-color: #b24c12;
+    outline: none;
+    image: url(skins:LateNight/palemoon/buttons/btn__delete_active.svg);
```

Comment:
Randomly stumbled upon this while building for iOS, this file seems to be missing, which results in a bunch of warnings of the form

```
qt.svg: warning Main Cannot open file '/private/var/containers/Bundle/Application/D577115B-62DB-4FFE-B072-4133259002FE/Mixxx.app/skins/LateNight/palemoon/buttons/btn__delete_active.svg', because: No such file or directory
```

(I know this PR didn't introduce this styling, but this is where the code was last touched)

Reply:
Yeah, I can also see this warning on Linux. Happy to create an issue? Should be an easy one to address.

Reply:
Done: #13492